src/app/shared/enums/storage.ts

```diff
@@ -15,6 +15,7 @@ export class StorageKeys {
   static SCHEDULE_VERSION = new StorageKeys('SCHEDULE_VERSION')
   static SCHEDULE_TASKS = new StorageKeys('SCHEDULE_TASKS')
   static SCHEDULE_TASKS_CLINICAL = new StorageKeys('SCHEDULE_TASKS_CLINICAL')
+  static SCHEDULE_TASKS_COMPLETED = new StorageKeys('SCHEDULE_TASKS_COMPLETED')
```

Comment:
Can we be more specific with this? Does this contain all the completed tasks? Or does this contain completed tasks from midnight? This is not evident from the name.

Reply:
Or maybe document this in here.

Reply:
It contains all completed tasks from midnight of last schedule generation date. Added documentation.